docs/quickstart/www/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
I think we can get rid of the `www` subdirectory altogether, as the `quickstart` directory does not contain anything else. This might need adjusting the README.md for the `php -S` commands to point to the right place.